src/infrastructure/database/modules/sqlite/repositories/sqlite-status-list.repository.ts

```
@@ -0,0 +1,228 @@
+      logger.error('Failed to delete status list in SQLite', {
+        error: error instanceof Error ? error.message : String(error),
+        statusListId: id
+      });
+      throw new Error(`Failed to delete status list: ${error instanceof Error ? error.message : String(error)}`);
+    }
+  }
+
+  /**
+   * Maps a database row to a StatusList entity
+   */
+  private mapRowToStatusList(row: any): StatusList {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)